### PR TITLE
Updated Capacity of "DE" zone

### PR DESCRIPTION
**Pull request by Bohne13**
updated capacity of "DE" zone. Data is coming from "Frauenhofer ISE" 
https://www.energy-charts.de/power_inst_de.htm?year=2020&period=annual&type=power_inst
last data update was on 1.09.20 at 22:55

### CODE REVIEW

**Review by FelixDQ — APPROVED** [review]
Thank you @Bohne13!

Can you also update the link to the data source in the readme?

Also, remember to add yourself in the list of contributors in the zone :-)

**Review by FelixDQ — CHANGES_REQUESTED** [review]
Woops. Waiting with merging until you have done the above :-)

File: config/zones.json [hunk 1]
```diff
@@ -1221,20 +1221,20 @@
...
-      "solar": 48570,
-      "unknown": 3137,
-      "wind": 60710
+      "solar": 51990,
+      "unknown": 3700,
+      "wind": 61880
     },
     "contributors": [
-      "https://github.com/corradio"
+      "https://github.com/bohne13"
```
Comment (FelixDQ) [inline config/zones.json:1237]: You should add yourself as an additional contributor :-)
```suggestion
      "https://github.com/corradio",
      "https://github.com/bohne13"
```

Comment (Bohne13) [reply config/zones.json:1237]: :D sorry, so now I was searching for the readme file to add the data source, I didn't found it!
I had this problem some time before :/

**Comment by FelixDQ** [comment]
No worries! It's in this file https://github.com/tmrowco/electricitymap-contrib#production-capacity-data-sources under `Production capacity sources`

**Review by FelixDQ — APPROVED** [review]

**Pull request merged**